CleanProFinder/CleanProFinder.Mobile/Views/InitialEditProfilePage.xaml

```diff
@@ -6,13 +6,19 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
     xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
     xmlns:viewmodel="clr-namespace:CleanProFinder.Mobile.ViewModels"
-    Title="InitialEditProfilePage"
-    x:DataType="viewmodel:InitialEditProfileViewModel">
+    Title="Initial Profile"
+    x:DataType="viewmodel:InitialEditProfileViewModel"
+    Background="{StaticResource Secondary}">
 
     <ContentPage.Resources>
         <ResourceDictionary>
             <toolkit:InvertedBoolConverter x:Key="InvertedBoolConverter" />
         </ResourceDictionary>
+        <Style x:Key="entryGreenStyle" TargetType="Entry">
```

Comment:
entryGreenStyle should start from capital letter

Reply:
(https://learn.microsoft.com/en-us/dotnet/maui/user-interface/styles/xaml#explicit-styles)